ddtrace/monkey.py

```diff
@@ -128,7 +141,7 @@ def _patch_module(module):
     """
     path = 'ddtrace.contrib.%s' % module
     with _LOCK:
-        if module in _PATCHED_MODULES and module != 'celery':
```

Comment:
for similar reasons to the above comment we should be sure to account for the module patched on import

Reply:
:+1: